manual/src/cmds/intf-c.etex

```diff
@@ -702,12 +702,20 @@ type) corresponds to the "Some" constructor.
 \subsection{ss:c-int-ops}{Operations on integers}
 
 \begin{itemize}
-\item "Val_long("\var{l}")" returns the value encoding the "long int" \var{l}.
-\item "Long_val("\var{v}")" returns the "long int" encoded in value \var{v}.
-\item "Val_int("\var{i}")" returns the value encoding the "int" \var{i}.
-\item "Int_val("\var{v}")" returns the "int" encoded in value \var{v}.
-\item "Val_bool("\var{x}")" returns the OCaml boolean representing the
-truth value of the C integer \var{x}.
+\item "Val_long("\var{n}")" returns the value encoding the C integer \var{n} of
+type "intnat". Despite its name, note that "intnat" is not necessarily the same
```

Comment:
There is no explicit documentation of `intnat` in this file. Could you maybe document this type on its own, before the list of operations?